lib/puppetserver/settings/ttl_setting.rb

```diff
@@ -0,0 +1,48 @@
+# A setting that represents a span of time to live, and evaluates to Numeric
```

Comment:
@Magisus not sure I understand your comment.  Is this something I can change now?  Yes, its copied (parts) from what we currently do in puppet repo

Reply:
No I think this is fine for now. I was just curious if you got it from somewhere, because this kind of parsing can be tricky, and I didn't want us to reimplement from scratch if we can copy from something that has been verified.